documentation/sdn/ovn_bgp_evpn_deployments.md

```diff
@@ -0,0 +1,696 @@
+# OVN BGP Agent in Kolla-Ansible for scalability and cloud interconnect deployments
+
+## Test deployment of OVN BGP Agent in Kolla-Ansible
+This part is a quick how-to for setting up OVN BGP Agent in Kolla-ansible.
+There's an existing [PR](https://review.opendev.org/c/openstack/kolla-ansible/+/891622) in kolla-ansible upstream, however it does not work properly. I created a [PR in kolla-ansible fork](https://github.com/SovereignCloudStack/kolla-ansible/pull/5) as well as in [kolla fork](https://github.com/SovereignCloudStack/kolla/pull/2) that attempts to fix the issues.
+
+I got all containers running, however they do not seem to activate  (see below)
+
+### Steps
+- Get a working test environment i.e. testbed, cloud-in-a-box, [kolla-builder](https://github.com/SovereignCloudStack/kolla-builder)
+
+ - If you use kolla-builder, you need to checkout a dev branch `ovn-bgp-agent-deployment`
+ ```bash
+ git fetch origin ovn-bgp-agent-deployment:ovn-bgp-agent-deployment
+ git checkout ovn-bgp-agent-deployment
+ ```
+ - Use the following variable file with kolla-builder
+```yaml
+inventory_name: "ovn-bgp"
+image_pool: "/home/matus/pool"
+image_path: "{{ image_pool }}/kolla-image.qcow2"
+ara_enable: false
+is_remote: false
+python_bin_path: "/home/matus/scs/kolla-venv/bin"
+# Use different CIDRs and virbrX interfaces if
+# the provided ones are already in use
+network_ssh: "kolla-ssh-bgp-flat"
+network_openstack: "kolla-openstack-bgp-flat"
+network_neutron: "kolla-neutron-bgp-flat"
+network_ssh_ip: "192.168.122.1"
+network_openstack_ip: "192.168.123.0"
+network_neutron_ip: "192.168.100.0"
+network_ssh_bridge: virbr0
+network_openstack_bridge: virbr1
+network_neutron_bridge: virbr2
+create_networks: true
+kolla_internal_vip_address: "192.168.123.200"
+git_patch_kolla: true
+git_branch: "ovn-bgp-agent"
+git_patches: []
+kolla_ssh_key: "{{ lookup('ansible.builtin.env', 'HOME') }}/.ssh/id_kolla"
+ssh_config: "ssh_config"
+globals_file: user-globals/ovn-bgp.yml
+kolla_source_local_path: "<path to kolla ansible source code>"
+is_aio: true
+node_name: "kolla-ovn-bgp-flat"
+ram_size: 10000
+cpu_count: 12
+
+```
+- Clone  [kola fork](https://github.com/SovereignCloudStack/kolla)
+```
+git clone git@github.com:SovereignCloudStack/kolla.git
+```
+>Note: the  steps regarding building kolla images are recommended the all-in-one VM spawned by kolla-builder rather than your local machine to simplyfy deployment.
```

Comment:
```suggestion
>Note: It is recommended to follow the steps for building Kolla images on the all-in-one VM created by kolla-builder, rather than on your local machine, to simplify the deployment process.
```